app/web/src/components/Hero/AssetWithGooey.tsx

```diff
@@ -32,40 +25,39 @@ export default function AssetWithGooey({
   const position = [offset[0], offset[1], 5]
   const scaleArr = Array.from({ length: 3 }).map(() => scale)
   const { mouse } = useThree()
+  const [rEuler, rQuaternion] = useMemo(
+    () => [new THREE.Euler(), new THREE.Quaternion()],
+    []
+  )
   useFrame((state, delta) => {
     if (edgeRef.current) {
       edgeRef.current.rotation.y += 0.01
-
-      coffeeRef.current.rotation.x = (-mouse.y * Math.PI) / 6
-      coffeeRef.current.rotation.y = (mouse.x * Math.PI) / 6
+    }
+    if (coffeeRef.current) {
+      rEuler.set((-mouse.y * Math.PI) / 4, (mouse.x * Math.PI) / 2, 0)
+      coffeeRef.current.quaternion.slerp(rQuaternion.setFromEuler(rEuler), 0.1)
```

Comment:
this makes the follow animation smoother by magic,

I just copy pasted it from an example so not fully on top of it, but I know that 'lerp' is linear interpolation, so uses that to gradually rotate the model.